src/bitcoin.rs

```diff
@@ -441,9 +420,9 @@ pub async fn fund_vault(
 
     Ok(FundVaultDetails {
         assets_output: Some(asset_outputs[0].to_owned()),
-        assets_change_output: Some(asset_outputs[1].to_owned()),
-        udas_output: Some(asset_outputs[2].to_owned()),
-        udas_change_output: Some(asset_outputs[3].to_owned()),
+        assets_change_output: None,
+        udas_output: Some(asset_outputs[1].to_owned()),
+        udas_change_output: None,
```

Comment:
I think the change outputs should be completely removed from the struct. Do you think that makes sense?
```suggestion
        udas_output: Some(asset_outputs[1].to_owned()),
```
It will need a matching change in typescript. It's just, if we're hardcoding that they're always none, then it makes sense to omit them entirely, right?

Reply:
Yes, I think make sense.

I will remove.